commcare_connect/templates/tailwind/base_table.html

```diff
@@ -1,113 +1,165 @@
+  <div class="scrollable-table-container">
+    <table class="scrollable-table">
+      {% block table.thead %}
+      {% if table.show_header %}
+      <thead>
+        <tr>
+          {% for column in table.columns %}
+          <th>
+            {% if column.orderable %}
+            <a href="{% querystring table.prefixed_order_by_field=column.order_by_alias.next %}" >
+              {{ column.header }}
+            </a>
+            {% else %}
+            <span>{{ column.header }}</span>
             {% endif %}
-            {% endblock table.thead %}
-            {% block table.tbody %}
-                <tbody {{ table.attrs.tbody.as_html }} class="bg-white divide-y divide-gray-200">
-                {% for row in table.paginated_rows %}
-                    {% block table.tbody.row %}
-                    <tr {{ row.attrs.as_html }}>
-                        {% for column, cell in row.items %}
-                            <td {{ column.attrs.td.as_html }} class="px-6 py-4 whitespace-nowrap">
-                                {% if column.localize == None %}
-                                    {{ cell }}
-                                {% else %}
-                                    {% if column.localize %}
-                                        {{ cell|localize }}
-                                    {% else %}
-                                        {{ cell|unlocalize }}
-                                    {% endif %}
-                                {% endif %}
-                            </td>
-                        {% endfor %}
-                    </tr>
-                    {% endblock table.tbody.row %}
-                {% empty %}
-                    {% if table.empty_text %}
-                    {% block table.tbody.empty_text %}
-                        <tr>
-                            <td colspan="{{ table.columns|length }}" class="px-6 py-4 text-center text-gray-500">
-                                {{ table.empty_text }}
-                            </td>
-                        </tr>
-                    {% endblock table.tbody.empty_text %}
-                    {% endif %}
-                {% endfor %}
-                </tbody>
-            {% endblock table.tbody %}
-            {% block table.tfoot %}
-            {% if table.has_footer %}
-            <tfoot {{ table.attrs.tfoot.as_html }} class="bg-gray-50">
-                <tr>
-                {% for column in table.columns %}
-                    <td {{ column.attrs.tf.as_html }} class="px-6 py-3 text-left text-xs font-medium text-gray-500">
-                        {{ column.footer }}
-                    </td>
-                {% endfor %}
-                </tr>
-            </tfoot>
-            {% endif %}
-            {% endblock table.tfoot %}
-        </table>
-    {% endblock table %}
+          </th>
+          {% endfor %}
+        </tr>
+      </thead>
+      {% endif %}
+      {% endblock table.thead %}
 
-    {% block pagination %}
```

Comment:
The pagination.html should be part of the base_table itself. Instead, I see that you have written raw pagination code separately for each page. And I don't see where you are using pagination.html file at all.